R/s3.get.R

```diff
@@ -59,11 +59,12 @@ s3.get <- function (path, bucket.location = "US", verbose = FALSE, debug = FALSE
     last_cached <- s3LRUcache()$last_accessed(path) # assumes a POSIXct object
 
     # Check time on s3 remote's copy
-    s3.cmd <- paste("ls ", path, "| head -n 1 | awk '{print $1\" \"$2}' ")
-    last_updated <- as.POSIXct(system2(s3cmd(), s3.cmd, stdout = TRUE), tz = "GMT")
+    s3.cmd <- paste("info ", path, "| head -n 3 | tail -n 1")
+    result <- system2(s3cmd(), s3.cmd, stdout = TRUE)
+    result <- substring(result, 20, 39)
```

Comment:
Some magic numbers here, but I don't expect the output format of `s3cmd info` to ever change.